monitoring/definitions/otel_collector.go

```diff
@@ -0,0 +1,169 @@
+package definitions
+
+import (
+	"github.com/sourcegraph/sourcegraph/monitoring/definitions/shared"
+	"github.com/sourcegraph/sourcegraph/monitoring/monitoring"
+	"time"
+)
+
+func OtelCollector() *monitoring.Dashboard {
+	containerName := "otel-collector"
+
+	return &monitoring.Dashboard{
+		Name:        containerName,
+		Title:       "OpenTelemetry Collector",
+		Description: "The OpenTelemetry collector ingests OpenTelemetry data from Sourcegraph and exports it to the configured backends.",
+		Groups: []monitoring.Group{
+			{
+				Title:  "Receivers",
+				Hidden: false,
+				Rows: []monitoring.Row{
+					{
+						{
+							Name:        "otel_span_receive_rate",
+							Description: "spans received per second",
+							Panel:       monitoring.Panel().Unit(monitoring.Number).LegendFormat("receiver: {{receiver}}"),
+							Owner:       monitoring.ObservableOwnerDevOps,
+							Query:       "sum(rate(otelcol_receiver_accepted_spans{receiver=~\"^.*\"}[1m])) by (receiver)",
```

Comment:
nit: can we put the `by` in front? I think this is the standard way to write this

```suggestion
							Query:       "sum by (receiver) (rate(otelcol_receiver_accepted_spans{receiver=~\"^.*\"}[1m]))",
```

Reply:
Agreed. Initially I had like that but then the generator failed ...

turns out I was missing a bracket FML 😭

Reply:
out of curiosity, was the error unhelpful? I was hoping adding query validation would improve the helpfulness here

Reply:
Also I believe you don't need `{receiver=~\"^.*\"}` here

Reply:
> out of curiosity, was the error unhelpful? I was hoping adding query validation would improve the helpfulness here

It certainly made me aware that I should go check the query in Grafana/Promotheus.

The whitespace check however ... 🥲 



> Also I believe you don't need `{receiver=~\"^.*\"}` here

I removed the internals but left the brackets ... which was on the fence about removing for some reason but as I typed this out I decided to remove them 😆